### ISSUE TITLE
Datatable: The content of each row in datatable is read as row in screen reader while navigating through keyboard

### ISSUE THREAD

**Issue by rinnuagna** [primefaces/primereact#8141]
### Describe the bug

When navigating through keyboard, screen reader is reading the content of table as row. Expecting to read the contents in each row

<img width="1572" height="758" alt="Image" src="https://github.com/user-attachments/assets/106b1423-6cb4-43b4-995a-d042e600c332" />

### System Information

```Shell
primereact 10.9.1
browser chrome
```

### Steps to reproduce the behavior

Create primereact table, read each row through screen reader 

### Expected behavior

The row contents should be read

**Comment by rinnuagna** [comment]
Any update on this

**Comment by melloware** [comment]
@rinnuagna if you need immediate support I suggest you look into PrimeReact PRO
